src/oralb_ble/parser.py

```diff
@@ -206,6 +211,7 @@ class ModelDescription:
     b'\x03"': Models.SmartSeries8000,
     b"\x03!": Models.SmartSeries9000,
     b"\x061": Models.IOSeries89,
+    b"\x062k": Models.IOSeries10,
```

Comment:
I don't think this will actually work. \x062k has a length of 3. When we use Bytes to model, we get the first 2 bytes. i.e. b"\x062" so it will still say it is Io Series 6/7.